app/models/ra/county_change.rb

```diff
@@ -1,3 +1,5 @@
 class Ra::CountyChange < ApplicationRecord
-  belongs_to :county_code, :class_name => 'Ra::CountyCode'
+  belongs_to :county_object, foreign_key: 'county_id', class_name: 'Ra::County'
```

Comment:
Su tam kolizie napr pri `Ra::StreetNameChange`, kde existuje atribut `street_name`, tak som to chcel mat konzistentne